### DISCUSSION TITLE
PGS SUM normalization

### DISCUSSION THREAD

**Discussion by cearring** [PGScatalog/pgsc_calc#378] [General]
Hello! I ran the pgs_calc pipeline with a downloaded scorefile from PGS Catalog (PGS002308) and my imputed data (from TopMED) and I am wondering if the PGS SUM that was calculated for my target data is normalized to the scorefile data or is just calculated raw based on the weights? The calculated PGS SUM distribution looks to have an almost normal distribution but the mean is ~0.3, so my guess is that the calculate PGS SUM data is not normalized yet, but I wanted to check out of caution. 
I am hoping to understand/confirm if the calculated PGS SUM can be interpreted relative to the publication the scorefile came from, ie if there is a way to determine which percentile our subjects are relative to the PGS distribution of the original paper.

**Comment by nebfield** [comment]
> Hello! I ran the pgs_calc pipeline with a downloaded scorefile from PGS Catalog (PGS002308) and my imputed data (from TopMED) and I am wondering if the PGS SUM that was calculated for my target data is normalized to the scorefile data or is just calculated raw based on the weights? The calculated PGS SUM distribution looks to have an almost normal distribution but the mean is ~0.3, so my guess is that the calculate PGS SUM data is not normalized yet, but I wanted to check out of caution.

PGS SUM is a raw value and not normalised, please see https://github.com/PGScatalog/pgsc_calc/discussions/294#discussioncomment-9506234 and [the docs which explain workflow outputs](https://pgsc-calc.readthedocs.io/en/latest/explanation/output.html#run-ancestry-specific-outputs)

> I am hoping to understand/confirm if the calculated PGS SUM can be interpreted relative to the publication the scorefile came from, ie if there is a way to determine which percentile our subjects are relative to the PGS distribution of the original paper.

I think that depends on how the distributions are reported in the original paper - what do you think @smlmbrt ?

**Reply by smlmbrt** [reply]
I think you would need the mean and sd from the original paper to do that. The accuracy of that sort of method would depend on having identical variant matching between the two populations (which is probably unlikely).
